Model/Request/Buyers.php

```diff
@@ -161,13 +166,25 @@ protected function getParams()
             'topic' => $this->_topic
         ];
 
-        $params['firstname'] = $this->session->getCustomer()->getFirstname() ?? '';
-        $params['lastname'] = $this->session->getCustomer()->getLastname() ?? '';
-        $params['email'] = $this->session->getCustomer()->getEmail() ?? '';
+        $params['firstname'] = $this->session->getCustomer()->getFirstname() ?? $this->params['first_name'];
+        $params['lastname'] = $this->session->getCustomer()->getLastname() ?? $this->params['last_name'];
+        $params['email'] = $this->session->getCustomer()->getEmail() ?? $this->params['email'];
```

Comment:
do we have other information that could be saved on the buyer?

Reply:
These are only basic things required while registration in magento.

Reply:
What other customer data is available? are there any docs of the customer entity?

Reply:
There's other data such as billing address, shipping address etc, but while registration we only get firstname lastname and email.